### PR TITLE
Update WCS remote data doctest

### LINKED ISSUE #9253: TST: WCS doctest fails with remote data
See https://travis-ci.org/astropy/astropy/jobs/586132282

```
________________________ [doctest] docs/wcs/wcsapi.rst _________________________

281 extracting a spectral slice, and in that slice we are extracting a sub-region

282 on the sky.

283 

284 If you are implementing your own WCS class, you could choose to implement

285 ``__getitem__`` and have it internally use

286 :class:`~astropy.wcs.wcsapi.SlicedLowLevelWCS`. In fact, the

287 :class:`~astropy.wcs.WCS` class does this - the example above can be written

288 more succinctly as::

289 

290     >>> wcs[10, 30:100, 30:100]  # doctest: +REMOTE_DATA

Differences (unified diff with -expected +actual):

    @@ -1,2 +1,3 @@

    +9

     SlicedFITSWCS Transformation

     <BLANKLINE>

    @@ -5,11 +6,11 @@

     Array shape (Numpy order): (70, 70)

     <BLANKLINE>

    -Pixel Dim  Data size  Bounds

    -        0         70  None

    -        1         70  None

    +Pixel Dim  Axis Name  Data size  Bounds

    +        0  None              70  None

    +        1  None              70  None

     <BLANKLINE>

    -World Dim  Physical Type  Units

    -        0  pos.eq.ra      deg

    -        1  pos.eq.dec     deg

    +World Dim  Axis Name  Physical Type  Units

    +        0  None       pos.eq.ra      deg

    +        1  None       pos.eq.dec     deg

     <BLANKLINE>

     Correlation between pixel and world axes:

/tmp/astropy-test-qsgze6b2/lib/python3.7/site-packages/docs/wcs/wcsapi.rst:290: DocTestFailure

====== 1 failed, 13457 passed, 112 skipped, 106 xfailed in 283.73 seconds ======
```

cc @astrofrog , @nden , and @Cadair

### PR DESCRIPTION
**Pull request by pllim**
Fix #9253 

Follow up of #9156 

Note to reviewer: Check the numpy-dev job that is allowed to fail.

### CODE REVIEW

**Comment by pllim** [comment]
What does this diff mean? 

```
290     >>> wcs[10, 30:100, 30:100]  # doctest: +REMOTE_DATA
Differences (unified diff with -expected +actual):
    @@ -1,2 +1,3 @@
    +9
     SlicedFITSWCS Transformation
     <BLANKLINE>
```

**Comment by aarchiba** [comment]
It seems to still fail the allowed-failures test?

**Comment by pllim** [comment]
> It seems to still fail 

Yes, it does. :sweat:

**Comment by bsipocz** [comment]
I think I manage to find a way to make the tests happy. Also rebased to make sure unrelated failures are not popping up.

File: docs/wcs/wcsapi.rst [hunk 1]
```diff
@@ -287,20 +287,22 @@ If you are implementing your own WCS class, you could choose to implement
 :class:`~astropy.wcs.WCS` class does this - the example above can be written
 more succinctly as::
 
-    >>> wcs[10, 30:100, 30:100]  # doctest: +REMOTE_DATA
+    >>> wcs[10, 30:100, 30:100]  # doctest: +REMOTE_DATA +ELLIPSIS
+    <BLANKLINE>
```
Comment (pllim) [inline docs/wcs/wcsapi.rst]: Makes no sense but I'll take the green status. Thank you so much, @bsipocz ! I can't approve my own PR, so feel free to approve it and merge. 🙏

Comment (bsipocz) [reply docs/wcs/wcsapi.rst]: I don't think that an ELLIPSIS can start a block, but this is just a wild guess. I tried the permutations until it worked 🤷‍♀

**Comment by bsipocz** [comment]
OK, let's merge this, we've beaten this 🐴 long enough.

File: docs/wcs/wcsapi.rst (hunk 1 above)
Comment (pllim) [reply docs/wcs/wcsapi.rst]: Thank you!!!

**Pull request merged**